CHANGES

```diff
@@ -1,5 +1,47 @@
 Revision history for Shaderc
 
+v2020.0 2020-01-17
+- glslc, libshaderc:
+  - Using SetBeforeHlslLegalization for more relaxed validation rules
+  - Added support for SPIR-V 1.5
+  - Add --emit-line-derective option
+  - Added support for Vulkan 1.2
+- spvc:
+  - Added option to inject robust buffer access code
+  - Added support for emitting ToVulkan shaders
+  - Added spirv-opt based IR generation as alternative to built in spirv_cross IR gen
+  - Added API for specifying source and target execution environments
+  - Added option & reflection API methods neede by Dawn
+  - Substantial internal refactoring and code cleanup
+  - Large number of breaking changes to the API
+    -  Replaced shaderc_spvc_compile_options_set_shader_model with
+    shaderc_spvc_compile_options_set_hlsl_shader_model
+    - Compiler initialization and shader generation moved into seperate calls
+    - Seperated return codes from shaderc ones
+    - Many small API changes
+  - Improvements to testing
+    - Refactoring and clean up of run_spirv_cross_tests.py
+    - Seperation of expectations into known failures, known invalids, and cases
+      that need investigation
+    - Tweaks and fixes to substantially increase passing cases
+    - Added support for running tests using spvc IR generation
+- Infrastructure:
+  - Removed Appveyor artifacts link
+  - Improvements and clean up of DEPS rolling scripts
+  - Enabled warnings about implicit fallthrough
+  - Enabled improper semicolon warnings
+  - Restricted -fPIC to platforms that support it
+  - Converted remaining scripts to use Python 3
+  - Replaced nosetest with unittest
+  - Removed assumptions aabout location of spirv-tools, effcee, and RE2
+  - Migrated BUILD.gn spirv_cross dependency to local repo
+- Fixes:
+  - Fixed BUILD.gn for Chromium, Dawn & Fuchsia
+  - Explicitly enabled HLSL support in glslang
+  - Added installing necessary headers in Android.mk
+  - Removed unsupported Unicode characters
+  - Fixed detection of spirv-headers directory
+
 v2019.1-dev 2019-06-04
```

Comment:
If you have finished with the v2019.1 release, you should remove the "-dev" from here.